src/scss/_elements.lists.scss

```diff
@@ -0,0 +1,46 @@
+
+ol:not([class]),
+ul:not([class]) {
+  margin: 1.5rem 0;
+}
+
+li:not([class]) {
+  margin-bottom: 0.75rem;
+}
+
+ol:not([class]) {
+  counter-reset: count;
+  list-style: none;
+
+  li:not([class]) {
+    counter-increment: count;
+    position: relative;
+
+    &::before {
+      content: counter(count)".";
+      color: $hotpink;
+      font-weight: bold;
+      display: inline-block;
+      position: absolute;
+      right: calc(100% + 1rem);
+      bottom: 0.075rem;
```

Comment:
This is pretty subjective feedback, but I'm wondering if we need this `bottom` property. The alignment looks strange to me, maybe because it's not big enough to look intentional? Seems like we could just have it aligned with the text. Up to you.